client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/EMController.java

```diff
@@ -507,6 +509,8 @@ public Response executeDatabaseCommand(DatabaseCommandDto dto, @Context HttpServ
 
         try {
 
+            ExecutionTracer.executingInitSql = true;
```

Comment:
unfortunately, I just realized this approach does not work with External Drivers :(  
we need to discuss this

Reply:
ok.